### ISSUE TITLE
Caso de estudio

### ISSUE THREAD

**Issue by yulydelgado** [JAPP-KING/CODE-KING#2]
[El Arca de los Sabores .pdf](https://github.com/user-attachments/files/17543016/El.Arca.de.los.Sabores.pdf)

**Comment by icoronado48-tech** [comment]
Hola Lista por aca

**Comment by JAPP-KING** [comment]
Listo por acá también

**Comment by luisgs2112** [comment]
Buenas tardes, conectado para comenzar con nuestra nueva reunion

**Comment by icoronado48-tech** [comment]
Falta dos compañeros?

**Comment by luisgs2112** [comment]
sip, pero mientras tanto podemos adelantar, quisiera aclarar una duda que tengo, al momento de realizar el caso de estudio, cada uno debera crear su base de datos y luego unirlos a github verdad? me falta descargar el Git que es una herramiento que segun lei, te permite conectar tu codigo con github

**Comment by JAPP-KING** [comment]
Bueno, mi gente vamos a comenzar, tenemos la siguiente división de temas: 

Luis Gañan: Gestion de Produccion

Andre Vergara: Gestion de Clientes

Jose A Peña: Gestion de Recetas

Jose D aliedes: Gestion de Inventario

Iliana Coronado: Documentación del Proyecto

**Comment by icoronado48-tech** [comment]
Hola Buena tardes, primero que nada les hago la salvedad que estoy conociendo github y comienzo a familiarizarme con esto. no se si hay una manera de hacer por aca un chat mas interactivo.

**Comment by JAPP-KING** [comment]
> sip, pero mientras tanto podemos adelantar, quisiera aclarar una duda que tengo, al momento de realizar el caso de estudio, cada uno debe crear su base de datos y luego unirlos a github verdad? me falta descargar el Git que es un herramiento que segun lei, te permite conectar tu codigo con github


Tengo entendido que cada quien programa su base de datos y luego unimos el proyecto en una base de datos madre, o podríamos hacer una base de datos madre y ahí trabajaríamos todos

**Comment by icoronado48-tech** [comment]
> > sip, pero mientras tanto podemos adelantar, quisiera aclarar una duda que tengo, al momento de realizar el caso de estudio, cada uno debe crear su base de datos y luego unirlos a github verdad? me falta descargar el Git que es un herramiento que segun lei, te permite conectar tu codigo con github
> 
> Tengo entendido que cada quien programa su base de datos y luego unimos el proyecto en una base de datos madre, o podríamos hacer una base de datos madre y ahí trabajaríamos todos.

Bueno segun entiendo git permite trabajar por ramas (branch)

**Comment by jos213** [comment]
Hola, José Aliendres conectado

**Comment by luisgs2112** [comment]
Bueno, yo he estado haciendo un curso extenso de postgresql aunque todavia no lo he aplicado al caso de estudio, por ahora las ideas iniciales que tengo es la creacion de una tabla con las siguientes caracteristicas.

En el caso de Planificación de producción estoy pensando en realizar esto:  Fecha de produccion, Id o nro de receta, tareas, personal (que seria el nombre de la persona), tiempo, cantidad a producir y queria agregar si se completo o no la tarea. 

Todavia estoy pensando en como hacer Seguimiento de la producción y Control de calidad pero no seria muy diferente a crear tablas, como todavia estoy estudiando postgresql veo que a nivel basico puedo hacer esto, por lo menos por ahora, aunque todavia me falta aplicarlo a la practica

**Comment by luisgs2112** [comment]
> r

correcto! primero pensemos en como vamos a diseñar las tablas

**Comment by icoronado48-tech** [comment]
Yo tengo algun material que nos puede servir, yo tengo un emprendimiento de postres y tengo creado en excel una tabla de productos y una ficha tecnica

**Comment by icoronado48-tech** [comment]
![image](https://github.com/user-attachments/assets/cd33c60a-0cde-4a0b-bc2e-bfc714badeee)

**Comment by icoronado48-tech** [comment]
![image](https://github.com/user-attachments/assets/c24350ee-d6fd-4497-a111-360e12498584)

**Comment by icoronado48-tech** [comment]
[Uploading Carta de alérgenos para restaurantes (Versión imprimible para rellenar).pdf…]()

**Comment by icoronado48-tech** [comment]
![image](https://github.com/user-attachments/assets/ce1188f2-cf6e-4fda-a8cb-151513984d52)

**Comment by icoronado48-tech** [comment]
De aca podemos sacar ideas para las tablas

**Comment by luisgs2112** [comment]
tienes razon amiga sobretodo para mi parte de gestion de produccion, pero es extraño, no lo puedo descargar

**Comment by icoronado48-tech** [comment]
> m
Bueno son print de pantallas, lo puedo compartir por telegram

**Comment by jos213** [comment]
Hola yo tengo algo de conocimiento de PostgreSQL, he estado investigando sobre el registro de inventario para la base de datos,¿ tengo una duda sobre agregar los equipos y mobiliarios ?  ya que va unido al departamento de compras donde se maneja las entrada y salida de mercancía.

**Comment by balthazar55** [comment]
Me parece excelente el PDF.

**Comment by balthazar55** [comment]
Ahora una pregunta. Trabajaremos todos en la misma base de datos media git o las haremos cada uno y luego las juntamos en git

**Comment by icoronado48-tech** [comment]
Les Propongo algo, vamos estudiando renglon por renglon del caso de estudio, y vamos estableciendo la estructura de cada tabla.

>

**Comment by balthazar55** [comment]
Me parece buena idea.

**Comment by luisgs2112** [comment]
por mi me parece bien amiga

**Comment by icoronado48-tech** [comment]
Ficha de receta Detallada: Creo lo podemos solucionar con la estructura que tengo, solo se podria agregar los alergenos creo que no lo contempla

**Comment by icoronado48-tech** [comment]
Biblioteca de sabores: Crear una biblioteca de sabores y aromas para facilitar la
creación de nuevas recetas y combinaciones. Esto creo son los ingredientes. Esto seria una tabla

**Comment by icoronado48-tech** [comment]
Gestión de alérgenos: Implementar un sistema para identificar y gestionar los
alérgenos presentes en cada plato. Esa tabla la resolvemos con el PDF y tengo otra información

**Comment by icoronado48-tech** [comment]
Menús temáticos: Facilitar la creación de menús temáticos personalizados para
eventos especiales o temporadas. Esto debo investigar un poco pero creo q es una tabla por tipo de menu

**Comment by jos213** [comment]
Estoy de acuerdo elaborando  Gestión de Recetas se procede hacer las tablas y base de datos de la gestión de inventario. ¿Cómo apoyo con recetas ? o vamos hacerla con cultura de varios Países .

**Comment by balthazar55** [comment]
Me parece buena idea. Lo de hacer varios países y varias recetas. Así colocamos campos extras que relacionar en las tablas

**Comment by icoronado48-tech** [comment]
Buenas Noche les comparto, la estructura de las tablas para su revisión y consideración. Esta estructurada de acuerdo al caso de estudio, solo faltan las consideraciones adicionales que toca el tema de E-COMERCE Y realidad aumentada. en relación a la tabla de fidelizacion tengo dudas en su estructura al igual que las tablas de ventas, que son las que  arrojarían los reportes. @JAPP-KING @yulydelgado @balthazar55 @Mauricio25Rod

**Comment by luisgs2112** [comment]
Buenas noches compañeros

**Comment by balthazar55** [comment]
Buenas noches.

**Comment by icoronado48-tech** [comment]
@balthazar55 Hola Buenas Noches

**Comment by balthazar55** [comment]
Tenemos que establecer los parámetros para crear la tabla y un diagrama de como se relacionarían las unas con la otras así todos trabajamos con el mismo esquema

**Comment by icoronado48-tech** [comment]
Totalmente de acuerdo

**Comment by icoronado48-tech** [comment]
[Tablas.pdf](https://github.com/user-attachments/files/17724441/Tablas.pdf)

**Comment by luisgs2112** [comment]
si no mal entiendo, seria indentificar las tablas principales no es asi? podemos empezar con las tablas de personal y recetas

**Comment by luisgs2112** [comment]
que serian una de las principales a mi entender

**Comment by icoronado48-tech** [comment]
Yo hice esto de acuerdo a lo que interprete pienso que puede ser un punto de partida para modificar y corregir, tengo entendido que la compañera programadora nos puede ayudar segun su experiencia

**Comment by balthazar55** [comment]
Si Yuli podría orientamos bien con eso. Ya que en eso trabaja

**Comment by icoronado48-tech** [comment]
creo que yuli no esta agregada a este grupo

**Comment by luisgs2112** [comment]
perfecto, en mi caso, como soy produccion las tablas que tengo asignado a crear son:

 personal, tareas, planificacion, seguimiento de produccion y control de calidad
 
  hay algunas mas que deba agregar para poder despues hacer la relacion? lo dejo aqui ir organizandolo

**Comment by icoronado48-tech** [comment]
bueno seria bueno que yuli revise e indique si esta bien estructurada las tablas y las relaciones de PK Y FK
